.github/dependabot.yml

```
@@ -4,25 +4,18 @@ updates:
     directory: "/"
     schedule:
-      interval: "daily"
-    labels:
-      - "type: dependencies"
+      interval: "weekly"
     commit-message:
       prefix: "ci"
-    rebase-strategy: disabled
-    assignees:
-      - "TomerFi"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)